pkgs/tools/security/crowbar/default.nix

```diff
@@ -0,0 +1,47 @@
+{ fetchFromGitHub
+, freerdp
+, nmap
+, openvpn
+, python3Packages
+, stdenv
+}:
+
+with python3Packages; buildPythonApplication rec {
+  pname = "crowbar";
+  version = "4.1";
+
+  disabled = !python3Packages.isPy3k;
+
+  src = fetchFromGitHub {
+    owner = "galkan";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1njbxgm7d0dih1fhqf2ycp74gf7w4kdw3rwmd4x0y8q4j6djh4yz";
+  };
+
+  propagatedBuildInputs = [ freerdp nmap openvpn python3Packages.paramiko ];
+
+  patchPhase = ''
+    cp ${./setup.py} setup.py
+    sed -i 's,/usr/bin/nmap,${nmap}/bin/nmap,' lib/nmap.py
+  '';
```

Comment:
this should be upstreamed, especially as a tool.

Then expose `entry_points` in the setup.py so that pip will create an executable as part of package installation. It will remove the need for your installPhase as well.